### PR TITLE
chore: [CHAOS-2427]: Added Release notes for 1.18.x release

### PR DESCRIPTION
**Pull request by Jonsy13**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by ajeshbaby — APPROVED** [review]
LGTM

**Comment by Jonsy13** [comment]
@KimberlyFields Can you please check this PR. Thanks!

File: release-notes/chaos-engineering.md [hunk 1]
```diff
@@ -18,10 +18,48 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
+* Now, Users will be provided with an error if there is already one experiment existing with the same name in ChaosHub while pushing an experiment to a ChaosHub. (CHAOS-872)
+
+* Vulnerability Enhancements - (CHAOS-2162)
+    - PromQL binary has been rebuilt with latest go1.20.7 & upgraded in chaos-go-runner docker image.
+    - Kubectl binary has been upgraded to v1.28.0 to reduce 2 vulnerabilities in K8s as well as chaos-go-runner docker image.
+    - Argo components like workflow-controller and argo-exec have been upgraded to v3.4.10 which resolves all vulnerabilities in respective components.
+
+### Early access
+
+* Linux Faults
```
Comment (KimberlyFields) [inline release-notes/chaos-engineering.md:38]: Early access is for features that are released behind a feature flag. Please include the name of the feature flag.

Comment (Jonsy13) [reply release-notes/chaos-engineering.md]: Thanks, have updated now!

**Review by KimberlyFields — APPROVED** [review]

**Pull request merged**